src/components/AssetsFilters/index.jsx

```diff
@@ -4,43 +4,50 @@ import CheckBoxGroup from '@edx/paragon/src/CheckBoxGroup';
 import CheckBox from '@edx/paragon/src/CheckBox';
 import { connect } from 'react-redux';
 
+import WrappedMessage from '../../utils/i18n/formattedMessageWrapper';
 import { filterUpdate } from '../../data/actions/assets';
 import styles from './AssetsFilters.scss';
+import messages from './displayMessages';
```

Comment:
What, if any, is the standard we use for ordering imports in jsx? It seems like we largely have third party and first party separate, do we go by <thing being imported> or <location we're importing from> after that?

Reply:
Not terribly important, just something I've been wondering

Reply:
@arizzitano might be able to speak more on what to do with our imports - I know there at least are _some_ eslint rules at play, but I'm not sure if there is a standard that we should be following?